debezium-core/src/main/java/io/debezium/pipeline/signal/SignalProcessor.java

```diff
@@ -74,9 +77,11 @@ public SignalProcessor(Class<? extends SourceConnector> connector,
         this.previousOffsets = previousOffsets;
         this.signalProcessorExecutor = Threads.newSingleThreadScheduledExecutor(connector, config.getLogicalName(), SignalProcessor.class.getSimpleName(), false);
 
-        signalChannelReaders.stream()
-                .filter(isEnabled())
-                .forEach(signalChannelReader -> signalChannelReader.init(connectorConfig));
+        // filter single channel reader based on configuration
+        this.enabledChannelReader = getEnabledChannelReaders();
```

Comment:
@TyrantLucifer just last one. Can you please rename it to `enabledChannelReaders`?

Reply:
Of course, my fault, sorry~